lib/services/resourceManagement/package.json

```diff
@@ -6,7 +6,7 @@
     "Zavery, Amar <amzavery@microsoft.com>",
     "Srinivasan, Vivek <visriniv@microsoft.com>"
   ],
-  "version": "3.1.1-preview",
+  "version": "3.2.1-preview",
```

Comment:
Since the API version changed, please do a major version bump. I also think we can remove the `-preview` suffix.
Also, did you update the version number in the azure-rest-api-specs repository too?

Reply:
Major version bump seems good--haven't changed anything in rest-api-specs. The automation there is also broken for mystery reasons which is why I manually opened this PR.

Reply:
>Major version bump seems good--haven't changed anything in rest-api-specs. The automation there is also broken for mystery reasons which is why I manually opened this PR.

The changes in the policy SDK are pretty significant. I don't know how versioning is done for this SDK, but my REST changes were to add a new [non-preview] API version for policy, so that should be reflected here using whatever versioning scheme is appropriate.

Reply:
Cool, I'll go with 4.0.0